test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTLSCertificateValidationTests.cs

```diff
@@ -29,15 +29,14 @@ public async Task DotnetRestore_withTLSCertificateValidationDisabled_DoesnotThro
         {
             // Arrange
             using var pathContext = _dotnetFixture.CreateSimpleTestPathContext();
-            using TestDirectory packageSourceDirectory = TestDirectory.Create();
             var packageA100 = new SimpleTestPackageContext("A", "1.0.0");
             await SimpleTestPackageUtility.CreateFolderFeedV3Async(
-                    packageSourceDirectory,
+                    pathContext.UserPackagesFolder,
```

Comment:
This is the global packages folder, did you intend to use that instead of the working directory and some specific subpath under that?